### PR TITLE
Finish up polkadot doc

### PR DESCRIPTION
**Pull request by juangirini**
Closes https://github.com/paritytech/polkadot-sdk-docs/issues/66

### CODE REVIEW

File: docs/sdk/src/polkadot_sdk/polkadot.rs [hunk 1]
```diff
@@ -83,5 +89,14 @@
 //!   Agile periodic-sale-based model for assigning Coretime on the Polkadot Ubiquitous Computer.
 //! - RFC#5: [Coretime-interface](https://github.com/polkadot-fellows/RFCs/blob/main/text/0005-coretime-interface.md):
 //!   Interface for manipulating the usage of cores on the Polkadot Ubiquitous Computer.
-// TODO: add more context and explanations about Polkadot as the Ubiquitous Computer and related
-// tech. https://github.com/paritytech/polkadot-sdk-docs/issues/66
+//!
+//! ### Polkadot as the Ubiquitous Computer
+//!
+//! - **Application Space**: for different parachains with their applications
```
Comment (gpestana) [inline docs/sdk/src/polkadot_sdk/polkadot.rs:95]:
```suggestion
//! - **Application Space**: for different parachains with their applications.
```

File: docs/sdk/src/polkadot_sdk/polkadot.rs [hunk 2]
```diff
@@ -83,5 +89,14 @@
...
+//! - **Accord Space**: for opt-in treaties across chains. Treaty's members cannot change/undermine
```
Comment (gpestana) [inline docs/sdk/src/polkadot_sdk/polkadot.rs:96]:
```suggestion
//! - **Accord Space**: for opt-in treaties across chains. Treaty's members cannot change/undermine.
```

(nits, but would be nice to add a period at the end of all items below)

File: docs/sdk/src/polkadot_sdk/polkadot.rs [hunk 3]
```diff
@@ -50,7 +50,8 @@
 //!   correct execution of all parachain, without having all of its validators re-execute all
 //!   parachain blocks. When seen from this perspective, the fact that Polkadot executes different
```
Comment (mrcnski) [inline docs/sdk/src/polkadot_sdk/polkadot.rs:51]:
```suggestion
//!   parachain blocks. When seen from this perspective, the fact that different Polkadot validators execute different
```

File: docs/sdk/src/polkadot_sdk/polkadot.rs [hunk 4]
```diff
@@ -50,7 +50,8 @@
...
 //!   parachains means it is a platform that has fully delivered (the holy grail of) "Full Execution
-//!   Sharding". TODO: link to approval checking article. https://github.com/paritytech/polkadot-sdk-docs/issues/66
+//!   Sharding".
+//!   Learn about [Approval Checking](https://docs.google.com/presentation/d/1fjiTeT33GpxeM9TyBvxFnSVYR9NiU9SrkA2RBxSL5k8)
```
Comment (mrcnski) [inline docs/sdk/src/polkadot_sdk/polkadot.rs:54]:
```suggestion
//!   Learn about this process called [Approval Checking](https://docs.google.com/presentation/d/1fjiTeT33GpxeM9TyBvxFnSVYR9NiU9SrkA2RBxSL5k8).
```

File: docs/sdk/src/polkadot_sdk/polkadot.rs [hunk 5]
```diff
@@ -83,5 +89,14 @@
...
-// tech. https://github.com/paritytech/polkadot-sdk-docs/issues/66
+//!
+//! ### Polkadot as the Ubiquitous Computer
+//!
+//! - **Application Space**: for different parachains with their applications.
+//! - **Accord Space**: for opt-in treaties across chains. Treaty's members cannot change/undermine
+//!   the treaty's logic. This allows trustless cross-chain applications.
+//! - **Core space-time**: for "consuming" coretime and using Polkadot computation. Collators send
+//!   blocks through cores to validators for validity and availability checks.
+//! - **Coretime**: sold through a broker system parachain.
```
Comment (mrcnski) [inline docs/sdk/src/polkadot_sdk/polkadot.rs:100]: These two points could be expanded/clarified a bit. Could also clarify the difference between core space-time and coretime.

File: docs/sdk/src/polkadot_sdk/polkadot.rs [hunk 6]
```diff
@@ -47,10 +47,11 @@
 //!   Chain*").
 //! * (heterogenous) Sharded Execution: Yet, each parachain is free to have its own execution logic
 //!   (runtime), which also encompasses governance and sovereignty. Moreover, Polkadot ensures the
-//!   correct execution of all parachain, without having all of its validators re-execute all
-//!   parachain blocks. When seen from this perspective, the fact that Polkadot executes different
-//!   parachains means it is a platform that has fully delivered (the holy grail of) "Full Execution
-//!   Sharding". TODO: link to approval checking article. <https://github.com/paritytech/polkadot-sdk-docs/issues/66>
+//!   correct execution of all parachains, without having all of its validators re-execute all
+//!   parachain blocks. When seen from this perspective, the fact that different Polkadot validators
```
Comment (gpestana) [inline docs/sdk/src/polkadot_sdk/polkadot.rs:51]:
```suggestion
//!   parachain blocks. When seen from this perspective, Polkadot achieves the ability to verify
//!   the validity of the block execution of multiple parachains using the same set of validators as
//!   the relay chain. In practice, this means that the shards (parachains) share the same economic
//!   security as the relay chain.
```

I think that something along these lines is less prone to confusion and more accurate.

File: docs/sdk/src/polkadot_sdk/polkadot.rs [hunk 7]
```diff
@@ -47,10 +47,11 @@
...
+//!   execute different parachains means it is a platform that has fully delivered (the holy grail
+//!   of) "Full Execution Sharding".
+//!   Learn about this process called [Approval Checking](https://docs.google.com/presentation/d/1fjiTeT33GpxeM9TyBvxFnSVYR9NiU9SrkA2RBxSL5k8).
```
Comment (gpestana) [inline docs/sdk/src/polkadot_sdk/polkadot.rs:54]: nit but maybe better to link also the talk (is there any?)

**Review by gpestana — CHANGES_REQUESTED** [review]

**Review by mrcnski — APPROVED** [review]

File: docs/sdk/src/polkadot_sdk/polkadot.rs [hunk 8]
```diff
@@ -83,5 +89,28 @@
...
+//! for resilient computation, offering a multicore computer-like environment where various
+//! parachains operate in parallel on different cores. This approach allows applications to access
+//! secure and decentralized computation flexibly and efficiently, marking a shift from a
+//! chain-centric to an application-centric ecosystem.
+//!
+//! #### We can visualize it as follows
+//! [![Polkadot as the Ubiquitous Computer](https://pbs.twimg.com/media/F3qCltlWkAECsWJ?format=jpg&name=medium)](https://twitter.com/filippoweb3/status/1692025249070563413)
+//! Let's break down the "space" layers from the above diagram:
+//! - **Application Space**: This is the space where each parachain's application logic lives as an
+//!   independent, but interconnected, blockchain.
```
Comment (joepetrowski) [inline docs/sdk/src/polkadot_sdk/polkadot.rs:107]: Sure, but this doesn't really say anything if you don't already understand it.

File: docs/sdk/src/polkadot_sdk/polkadot.rs [hunk 9]
```diff
@@ -83,5 +89,28 @@
...
+//! - **Accord Space**: Is the space for opt-in treaties across chains. Treaty's members cannot
+//!   change/undermine the treaty's logic. This allows trustless cross-chain applications.
```
Comment (joepetrowski) [inline docs/sdk/src/polkadot_sdk/polkadot.rs:109]:
```suggestion
//! - **Accord Space**: Is the space for opt-in treaties across chains. A treaty's members cannot
//!   change/undermine the treaty's logic. This allows trustless cross-chain applications.
```

File: docs/sdk/src/polkadot_sdk/polkadot.rs [hunk 10]
```diff
@@ -83,5 +89,28 @@
...
+//! - **Core Space-time**: This can be seen as the CPU of the Polkadot Ubiquitous Computer, where
+//!   the processes are queued and executed. In oder words, we can say that the *Core Space-time*
+//!   "consumes" *Coretime* using Polkadot computation. Here is where collators send blocks through
+//!   cores to validators for validity and availability checks.
+//!
+//! Additionally, this introduces the concept of **Coretime**. Coretime is the right to use
+//! Polkadot's computing power. Sold through a broker system parachain.
```
Comment (joepetrowski) [inline docs/sdk/src/polkadot_sdk/polkadot.rs:116]: 🤮  Sorry but now this is just throwing around a bunch of terms and makes it more confusing, not more clear.

File: docs/sdk/src/polkadot_sdk/polkadot.rs [hunk 11]
```diff
@@ -83,5 +89,28 @@
...
+//! beyond its initial concept into a platform focusing on blockspace (the fundamental resource
+//! chains need) rather than the chains themselves. Polkadot serves as a general-purpose provider
+//! for resilient computation, offering a multicore computer-like environment where various
+//! parachains operate in parallel on different cores. This approach allows applications to access
+//! secure and decentralized computation flexibly and efficiently, marking a shift from a
+//! chain-centric to an application-centric ecosystem.
+//!
+//! #### We can visualize it as follows
+//! [![Polkadot as the Ubiquitous Computer](https://pbs.twimg.com/media/F3qCltlWkAECsWJ?format=jpg&name=medium)](https://twitter.com/filippoweb3/status/1692025249070563413)
+//! Let's break down the "space" layers from the above diagram:
```
Comment (joepetrowski) [inline docs/sdk/src/polkadot_sdk/polkadot.rs:105]: I understand this diagram may have been used in some presentations, but it doesn't mean we need to reuse it here. It's a different medium than being able to explain something in speech alongside it. IMO this whole section reduces clarity.

File: docs/sdk/src/polkadot_sdk/polkadot.rs (hunk 6 above)
Comment (joepetrowski) [reply docs/sdk/src/polkadot_sdk/polkadot.rs:51]: But capitalize Relay Chain.

File: docs/sdk/src/polkadot_sdk/polkadot.rs (hunk 7 above)
Comment (juangirini) [reply docs/sdk/src/polkadot_sdk/polkadot.rs:54]: I couldn't found any... maybe @rphmeier knows?

Comment (ordian) [reply docs/sdk/src/polkadot_sdk/polkadot.rs:54]: I'd suggest this comprehensive post by Rob:
https://polkadot.network/blog/polkadot-v1-0-sharding-and-economic-security/#approval-checking-and-finality

File: docs/sdk/src/polkadot_sdk/polkadot.rs [hunk 12]
```diff
@@ -83,5 +89,28 @@
...
+//!
+//! ### Polkadot, a Ubiquitous Computer
+//!
+//! Polkadot can be described as a "ubiquitous computer" based on its architecture. It is evolving
+//! beyond its initial concept into a platform focusing on blockspace (the fundamental resource
+//! chains need) rather than the chains themselves. Polkadot serves as a general-purpose provider
+//! for resilient computation, offering a multicore computer-like environment where various
+//! parachains operate in parallel on different cores. This approach allows applications to access
+//! secure and decentralized computation flexibly and efficiently, marking a shift from a
+//! chain-centric to an application-centric ecosystem.
```
[the remark this answers is not in the document]
Comment (juangirini) [reply docs/sdk/src/polkadot_sdk/polkadot.rs]: based on DRY principle, I have decided to just link to the wiki

**Review by joepetrowski — APPROVED** [review]

**Review by gpestana — APPROVED** [review]

**Review by michalkucharczyk — APPROVED** [review]

**Review by ordian — APPROVED** [review]

**Review by seadanda — APPROVED** [review]

**Pull request merged**